tests/e2e/pageobjects/ide/DebugView.ts

```diff
@@ -52,12 +52,25 @@ export class DebugView {
     async waitForDebuggerToConnect(timeout: number = TimeoutConstants.TS_DEBUGGER_CONNECTION_TIMEOUT) {
         await this.driverHelper.getDriver().wait(async () => {
             Logger.debug(`Waiting for debugger to connect (threads to appear in "Threads" view)`);
-            const threadElements: WebElement[] = await this.driverHelper.getDriver().findElements(By.xpath(`//div[contains(@class, 'theia-debug-thread')]`));
+
+            const threadsTreeLocator = `//div[contains(@class, 'theia-debug-thread')]`;
+
+            const threadElements: WebElement[] = await this.driverHelper.getDriver().findElements(By.xpath(threadsTreeLocator));
```

Comment:
maybe simplified by ```driverHelper.waitAllPresence```

Reply:
thanks for notice!
It has been applied.

Reply:
thanks for notice!
It has been applied.